metadata-db-app/src/cmr/metadata_db/migrations/086_remove_mimetype_generics_table.clj

```diff
@@ -0,0 +1,17 @@
+(ns cmr.metadata-db.migrations.086-remove-mimetype-generics-table
+   (:require
+   [config.mdb-migrate-helper :as h]))
+
+(defn up
+  "Migrates the database up to version 86."
+  []
+  (println "cmr.metadata-db.migrations.086-remove-mimetype-generics-table up...")
+  (h/sql "ALTER TABLE CMR_GENERIC_DOCUMENTS ADD MIME_TYPE VARCHAR(255)")
```

Comment:
The migration code is the opposite of what should have happened. It is adding the MIME_TYPE column instead of removing it.

Reply:
correct.